### PR TITLE
Improved Action Error Msg

### LINKED ISSUE #152: [BUG] Improve Error Message about Defining Video First
**Describe the bug**

An expected error appears when you do not define a `Video` object before a list of Actions. This is expected, but the error string could be improved as to why this occurs.

**To Reproduce**

1. Julia Version (i.e. output of `julia -v`): 1.5

2. Operating system (Mac, Linux, Windows): Linux (Fedora 32)

3. Minimum working code example that led to bug:

```julia
action_list = [Action(...), Action(...), Action(...)]
myvideo = Video(...)
javis(myvideo, action_list, "test.gif")
```

**Expected Behavior and Actual Behavior**

I didn't expect an error initially when ordering the code this way, but once I understood that `Action` objects implicitly require a `Video` object before an `Action` list, it makes sense.

**Stacktrace (If Applicable)**

```                                                                                    
ERROR: LoadError: BoundsError: attempt to access 0-element Array{Video,1} at index [1] 
Stacktrace:                                                                            
 [1] getindex at ./array.jl:809 [inlined]                                              
 [2] Action(::UnitRange{Int64}, ::Nothing, ::Function; kwargs::Base.Iterators.          Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/src/Projects/javis/    src/Javis.jl:397                                                                      
 [3] Action at /home/src/Projects/javis/src/Javis.jl:397 [inlined]                     
 [4] #Action#3 at /home/src/Projects/javis/src/Javis.jl:350 [inlined]                  
 [5] Action(::UnitRange{Int64}, ::Function) at /home/src/Projects/javis/src/Javis.jl:   350                                                                                   
 [6] top-level scope at /home/src/Projects/javis/src/javis_viewer.jl:34                
 [7] include(::String) at ./client.jl:457                                              
 [8] top-level scope at REPL[1]:1                                                      
in expression starting at /home/src/Projects/javis/src/javis_viewer.jl:34              
```       

**Additional context**

Found while working on #119

### PR DESCRIPTION
**Pull request by TheCedarPrince**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [x] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [x] Did I make sure to only change the part of the file where I introduced a new change/feature?
- [x] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [x] Did I properly add Javis dependencies to the `Project.toml` + set an upper bound of the dependency (if applicable)?
- [x] Did I properly add test dependencies to the `test` directory (if applicable)?
- [x] Did I check relevant tutorials that may be affected by changes in this PR?
- [x] Did I clearly articulate why this PR was made the way it was and how it was made?

**Link to relevant issue(s)**
Closes #152

**How did you address these issues with this PR? What methods did you use?**

I used the `ErrorException` message to raise a message to the user about defining a `Video` before an `Action` object.

### CODE REVIEW

File: src/Javis.jl [hunk 1]
```diff
@@ -537,24 +537,28 @@ function Action(
...
 )
-    CURRENT_VIDEO[1].defs[:last_frames] = frames
-    opts = Dict(kwargs...)
-    subactions = SubAction[]
-    if haskey(opts, :subactions)
-        subactions = opts[:subactions]
-        delete!(opts, :subactions)
+    if isempty(CURRENT_VIDEO)
+        throw(ErrorException("A `Video` must be defined before an `Action`"))
+    else
```
Comment (Wikunia) [inline src/Javis.jl:542]: `throw` doesn't need an else :wink:

**Review by Wikunia — CHANGES_REQUESTED** [review]
Besides the comment:
Do you mind creating a unit test for this?

**Review by TheCedarPrince — COMMENTED** [review]
@Wikunia, the unit test is functional now and I removed the else statement from the throw expression. Ready for your review.

**Review by Wikunia — APPROVED** [review]
LGTM :+1:

**Pull request merged**